submodules/externals/antlr3

```diff
@@ -1 +1 @@
-Subproject commit 311b6764e3f440db43617296ff856f7ec07bfdef
+Subproject commit 812695383316c7227c06df2b30725a9e5e4a11a0
```

Comment:
Looking at https://github.com/BelledonneCommunications/linphone-iphone, this is 311b6764e3f440db43617296ff856f7ec07bfdef, why would I want to use 812695383316c7227c06df2b30725a9e5e4a11a0?